assignment-client/src/avatars/AvatarMixer.cpp

```diff
@@ -259,8 +259,8 @@ void AvatarMixer::broadcastAvatarData() {
                         return;
                     }
 
-                    PacketSequenceNumber lastSeqToReceiver = nodeData->getLastBroadcastSequenceNumber(otherNode->getUUID());
-                    PacketSequenceNumber lastSeqFromSender = otherNode->getLastSequenceNumberForPacketType(PacketType::AvatarData);
+                    uint16_t lastSeqToReceiver = nodeData->getLastBroadcastSequenceNumber(otherNode->getUUID());
```

Comment:
It no longer applies to all nodes and node types - I moved the sequence numbering we were doing for Avatars to be specific to Avatars. Could add a new type def if you think it'll be cleaner.